internal/provider/access_level_helpers.go

```diff
@@ -64,6 +64,14 @@ var validProjectEnvironmentStates = []string{
 	"available", "stopped",
 }
 
+var validGroupSamlLinkAccessLevelNames = []string{
```

Comment:
I think it would be okay to move those into the `internal/provider/resource_gitlab_group_saml_link.go` file, given that it's only used there.